packages/ui/src/react-form/form-item.tsx

```diff
@@ -11,7 +11,7 @@ export const FormItemContext = React.createContext<FormItemContextValue>(
   {} as FormItemContextValue
 );
 
-interface FormItemProps extends FormItemWrapperProps {}
+export interface FormItemProps extends FormItemWrapperProps {}
```

Comment:
Add it in a common place if it is going to be used in multiple places

Reply:
No @yaju09 . This is just for the building of the ui package. There is no use case for this in codebase